web/src/features/filtering/components/ObserverStatusSelect.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { useFilteringContainer } from '../hooks/useFilteringContainer';
+import { GenericSelectFilter, GenericSelectFilterOption } from './GenericSelectFilter';
+
+const observerStatusOptions: GenericSelectFilterOption[] = [
+  {
+    value: 'Active',
+    label: 'Active',
+  },
+
+  {
+    value: 'Pending',
```

Comment:
please use `MonitoringObserverStatus` from `~/src/features/monitoring-observers/models/monitoring-observer.ts`
if needed you can move the types in a better suited place